packages/js/src/types.ts

```diff
@@ -54,6 +54,13 @@ export type MatchLogic =
   | { regex: string }
   | { starts_with: string }
 
+export interface FindItem {
```

Comment:
if I understand the usage of this correctly, this looks more like
```ts
interface ArrayMatch {
  field: string
  itemMatch: MatchLogic
}
```

Reply:
done.